python/uv/_build_backend.py

```diff
@@ -26,14 +26,15 @@ def warn_config_settings(config_settings: "dict | None" = None):
 
 def call(args: "list[str]", config_settings: "dict | None" = None) -> str:
     """Invoke a uv subprocess and return the filename from stdout."""
+    import shutil
     import subprocess
     import sys
 
-    from ._find_uv import find_uv_bin
-
     warn_config_settings(config_settings)
+    # Unlike `find_uv_bin`, this mechanism must work according to PEP 517
+    uv_bin = shutil.which("uv")
```

Comment:
if for some reason, uv isn't (properly) installed or can't be found on the path, `shutil.which` returns none. The following can guard against this and give a bit of feedback:

```suggestion
    uv_bin = shutil.which("uv")
    if uv_bin is None:
        raise RuntimeError("uv was not properly installed")
```

this should also address the type checking error in CI

Reply:
Good idea